lib/ansible/modules/system/known_hosts.py

```diff
@@ -119,12 +119,15 @@ def enforce_state(module, params):
 
     found,replace_or_add,found_line,key=search_for_host_key(module,host,key,hash_host,path,sshkeygen)
 
+    params['diff'] = compute_diff(path, found_line, replace_or_add, state, key)
```

Comment:
Imo `params` is not the right place for this. Since you once use the content here, why not call it `diff`?